plasma_framework/migrations/1041_setup_vault.js

```diff
@@ -40,45 +40,55 @@ const waitForReceipt = async (whatLog, transaction) => {
     console.log(`Submitted transaction with hash for ${whatLog}: ${transaction.transactionHash}`);
+    console.log(`Full log for ${whatLog}: ${transaction}`);
     await waitForReceipt(whatLog, transaction);
 };
 
 const paymentExitGameInit = async (whatLog, paymentExitGame, gnosisMultisigAbi, gnosisMultisigAddress, deployerAddress) => {
     const paymentExitGameInitCall = web3.eth.abi.encodeFunctionCall(paymentExitGame.abi.find(o => o.name === 'init'), []);
     const gnosisPaymentExitGameInit = web3.eth.abi.encodeFunctionCall(gnosisMultisigAbi, [paymentExitGame.address, 0, paymentExitGameInitCall]);
-    const gas = await web3.eth.estimateGas({ to: gnosisMultisigAddress, from: deployerAddress, data: gnosisPaymentExitGameInit });
+    const gas = await web3.eth.estimateGas({ to: gnosisMultisigAddress, from: deployerAddress, data: gnosisPaymentExitGameInit }) * 2;
+    console.log(`The amount of gas used for ${whatLog}: ${gas}`);
     const transaction = await web3.eth.sendTransaction({ gas: gas, to: gnosisMultisigAddress, from: deployerAddress, data: gnosisPaymentExitGameInit });
     console.log(`Submitted transaction with hash for ${whatLog}: ${transaction.transactionHash}`);
+    console.log(`Full log for ${whatLog}: ${transaction}`);
     await waitForReceipt(whatLog, transaction);
 };
 
 const registerExitGame = async (whatLog, plasmaFramework, txType, exitGame, id, gnosisMultisigAbi, gnosisMultisigAddress, deployerAddress) => {
     const registerFeeExitGameCall = web3.eth.abi.encodeFunctionCall(plasmaFramework.abi.find(o => o.name === 'registerExitGame'), [txType, exitGame.address, id]);
     const gnosisFeeRegisterExitGame = web3.eth.abi.encodeFunctionCall(gnosisMultisigAbi, [plasmaFramework.address, 0, registerFeeExitGameCall]);
-    const gas = await web3.eth.estimateGas({ to: gnosisMultisigAddress, from: deployerAddress, data: gnosisFeeRegisterExitGame });
+    const gas = await web3.eth.estimateGas({ to: gnosisMultisigAddress, from: deployerAddress, data: gnosisFeeRegisterExitGame }) * 2;
+    console.log(`The amount of gas used for ${whatLog}: ${gas}`);
     const transaction = await web3.eth.sendTransaction({ gas: gas, to: gnosisMultisigAddress, from: deployerAddress, data: gnosisFeeRegisterExitGame });
     console.log(`Submitted transaction with hash for ${whatLog}: ${transaction.transactionHash}`);
+    console.log(`Full log for ${whatLog}: ${transaction}`);
     await waitForReceipt(whatLog, transaction);
 };
 
 const setVersion = async (whatLog, plasmaFramework, sha, gnosisMultisigAbi, gnosisMultisigAddress, deployerAddress) => {
     const setVersionCall = web3.eth.abi.encodeFunctionCall(plasmaFramework.abi.find(o => o.name === 'setVersion'), [`${pck.version}+${sha}`]);
     const gnosisSetVersion = web3.eth.abi.encodeFunctionCall(gnosisMultisigAbi, [plasmaFramework.address, 0, setVersionCall]);
     const gas = await web3.eth.estimateGas({ to: gnosisMultisigAddress, from: deployerAddress, data: gnosisSetVersion });
```

Comment:
do we not need `*2` for this one?

Reply:
not sure, but at the moment even if it's reverted, there's no impact. the version I'll build after this PR will handle all these cases.